spec/system/provider_interface/provider_signs_in_from_new_device_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe 'A provider authenticates via DfE Sign-in from two separate devices' do
+  include DfESignInHelpers
+
+  let(:provider_user) { create(:provider_user, email_address: 'provider@example.com', dfe_sign_in_uid: 'DFE_SIGN_IN_UID', first_name: 'Michael') }
+
+  scenario 'signing in successfully from two different devices' do
+    given_i_am_registered_as_a_provider_user
+    and_i_have_a_dfe_sign_in_account
+
+    when_i_visit_the_provider_interface_applications_path
+    then_i_am_redirected_to_the_provider_sign_in_path
+
+    when_i_sign_in_via_dfe_sign_in
+    then_i_am_redirected_to_the_provider_interface_applications_path
+
+    when_i_click_sign_out
+    and_sign_in_from_a_different_device
+    then_i_receive_a_confirmation_email_with_correct_details
```

Comment:
I'd give it one more turn of the handle here — if you then come back and sign in with the same device, do you get another email?